src/index.js

```diff
@@ -0,0 +1,38 @@
+import './style.css';
+import Img from './assets/rotateArrow.png';
+import enter from './assets/enter.png';
+
+const tasksContainer = document.getElementById('tasks');
+const rotateArrow = document.getElementById('rotate-arrow');
+const enterIcon = document.getElementById('enter-icon');
+// const titleDiv = document.querySelector('.title');
+rotateArrow.src = Img;
+enterIcon.src = enter;
+
+const tasks = [
+  {
+    description: 'went to market',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'finish my project',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'washing clothes',
+    completed: false,
+    index: 2,
+  },
+];
+
+//  <div class="each-list-container"></div>
```

Comment:
* Kindly remove all commented code from your codebase as it makes your code look bulky and unprofessional 🙏

Reply:
Ok, thank you. I will walk through it.